react/week2/my-app/src/render.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+const list = [
+  {
+    id: 1,
+    description: "Get out of bed"
+  },
+  {
+    id: 2,
+    description: "Brush teeth"
+  },
+  {
+    id: 3,
+    description: "Eat breakfast"
+  },
+
+  {
+    id: 4,
+    description: "Go for a walk"
+  },
+
+  {
+    id: 5,
+    description: "Finish homework "
+  }
+];
+function RenderList(props) {
```

Comment:
I would consider renaming this to simply `TodoItem`. 

When naming components you should think in terms of what they are, not how it's implemented.